.github/workflows/python.yml

```diff
@@ -15,7 +15,7 @@ on:
       - .github/workflows/run_tests/**
 jobs:
   linux:
-    timeout-minutes: 30
+    timeout-minutes: 45
```

Comment:
i thought the long poll is duckdb and not the python gha?

Reply:
There are a few create index tests seem slow on GHA.
https://github.com/eto-ai/lance/actions/runs/4493433922